packages/manager/modules/pci/src/projects/project/storages/cold-archive/users/translations/Messages_fr_FR.json

```diff
@@ -13,6 +14,6 @@
   "pci_projects_project_storages_containers_users_add_user": "Ajouter un utilisateur",
   "pci_projects_project_storages_containers_users_import_success_message": "Le téléchargment du fichier JSON lié à l’utilisateur {{user}} a été effectué avec succès.",
   "pci_projects_project_storages_containers_users_import_error_message": "Une erreur est survenue lors du téléchargement du fichier JSON lié à l’utilisateur : {{ message }}.",
-  "pci_projects_project_storages_containers_users_show_secret_key_success": "La clé secrète de votre utilisateur {{user}} est {{secret}}.",
+  "pci_projects_project_storages_containers_users_show_secret_key_success": "La clé secrète de votre utilisateur <strong>{{user}}</strong> est {{secret}}.",
```

Comment:
**nitpick:** `<strong>` html tag can be avoided in translation files by including it in the `user` in HTML with `data-translate-value`

Reply:
Done